### PR TITLE
Correction of "Redirect URI" in setup

### PR DESCRIPTION
**Pull request by kevinholtkamp**
Added that your server needs to be publicly available for the setup to work

## Proposed change

Added that your server needs to be publicly available for the setup to work

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

None needed

## Checklist

- [x ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @kevinholtkamp, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19029).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/home_connect.markdown [hunk 1]
```diff
@@ -43,6 +43,7 @@ Note that it depends on the appliance and on API permissions which of the featur
 - OAuth Flow: Authorization Code Grant Flow
 - Redirect URI: "`<INTERNAL_HOME_ASSISTANT_URL>/auth/external/callback`
   Use your internal Home Assistant URL, if you didn't configure one manually, use your local IP address. Examples: `http://192.168.0.2:8123/auth/external/callback`, `http://homeassistant.local:8123/auth/external/callback`."
+- If your Home Assistant server is not reachable from the internet, you need to also add your public IP address in the same format as above and forward the Port in your router settings. The port can be closed after the setup is completed.
```
Comment (frenck) [inline source/_integrations/home_connect.markdown:46]: This is OAuth, it should not need public (as per OAuth specifications).

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/home_connect.markdown (hunk 1 above)
Comment (kevinholtkamp) [reply source/_integrations/home_connect.markdown:46]: I tried it using only my local Home Assistant address, but when i tried to setup the integration, it forwarded me to https://api.home-connect.com/security/oauth/login, where i got an error message saying something along the lines of "unauthorized host". I setup the port forwarding and entered my public IP as a second "Redirect URI" in the Home Connect Application, it worked perfectly.

Comment (frenck) [reply source/_integrations/home_connect.markdown:46]: Ok, but that sounds weird, as that would mean the instructions above are incorrect?

**Comment by frenck** [comment]
Since there was no response at all, I'm declaring this PR stale and closing it.
Feel free to re-open this PR when you feel like that is the appropriate thing to do.

**Pull request closed without merging**